source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -126,172 +130,137 @@ used at the data preparation and data processing stages for the following:
-  the original dataset stored row-by-row (there are strides between values of a
-  single feature).
-
-- To acquire data in a desired :capterm:`data format` for which
+- Acquire data in a desired :capterm:`data format` for which
   a specific set of operations is defined.
 
-- To have read-only, read-write and write-only access to the data. Accessor
+- Have read-only, read-write and write-only access to the data. Accessor
   implementations are not required to have read-write and write-only access
   modes for :capterm:`immutable <immutability>` entities like
   :txtref:`<table>s`.
 
 For details, see :txtref:`accessors` section.
 
-Use-case example for table, accessor and table builder
-------------------------------------------------------
-
-This section provides a basic usage scenario of the :txtref:`table`,
-:txtref:`table-builder`, and :txtref:`accessor` concepts and demonstrates the
-relations between them. :txtref:`The following diagram
-<data_management_sequence_diagram>` shows objects of these concepts, which are
-highlighted by colors:
+Example of interaction between table and accessor objects
+---------------------------------------------------------
 
-- :txtref:`Table builder <table-builder>` objects are blue.
+This section provides a basic usage scenario of the :txtref:`table` and
+:txtref:`accessor` concepts and demonstrates the relations between them.
+:txtref:`The following diagram <table_accessor_usage_example>` shows objects
+of these concepts, which are highlighted by colors:
 
-- :txtref:`Table <table>` objects are cyan.
+- :txtref:`table` object is dark blue
 
-- :txtref:`Accessors <Accessor>` are yellow.
+- :txtref:`accessor` is orange
 
-- Grey objects are not a part of |dal_short_name| specification and they are
-  provided just for illustration purposes.
+- :txtref:`table_metadata` is light blue
 
-.. _data_management_sequence_diagram:
+.. _table_accessor_usage_example:
 
-.. image:: _static/table_builder_accessor_sequence.png
+.. image:: _static/table_accessor_usage_example.png
   :width: 800
   :alt: Sequence diagram of accessor-builder-table relations
 
 To perform computations on a dataset, one shall create a :txtref:`table` object
-first. It can be done using a :txtref:`data-source` or a :txtref:`table-builder`
-object depending on the situation. The diagram briefly shows the situation when
-:txtref:`table` is interactively created from a various external entities (not
-shown on a diagram) using a :txtref:`table-builder`.
+first. It can be done either using a :txtref:`data-source` or directly from
+user-defined memory. The diagram shows the creation of a :txtref:`table` object
+`t` from the data provided by user (not shown on a diagram). During a table
+creation, an object `tm` of table metadata is created and its content is deduced
+from the incoming data.
 
 Once a table object is created, the data inside it can be accessed by its own
```

Comment:
I'd swap sentences
Once a table object is created, it can be used as an input...
The data in the table can be accessed via interface provided by the table or via read-only accessor as shown on the diagram